datajs/src/parser/csv.js

```diff
@@ -25,10 +25,9 @@ export async function guessParseOptions(file) {
   if (file.displayName === 'FileLocal') {
     const stream = await file.stream({ end: 50000 })
     text = await toString(stream)
-
   } else if (file.displayName === 'FileInterface') {
-    text = await file.descriptor.text()
-
+    // text = await file.descriptor.text()
+    text = await getChunkText(file)
```

Comment:
@risenW Nice replacement  👍 Can we delete the commented function?

Reply:
Sure